modules/microshift-disabling-lvms-csi-snapshot.adoc

```diff
@@ -32,11 +32,11 @@ This procedure is for users who are defining the configuration file before insta
 <1> Accepted values are:
 * Not defining `optionalCsiComponents`.
 * Specifying `optionalCsiComponents` field with an empty value (`[]`) or a single empty string element (`[""]`).
-* Specifying `optionalCsiComponents` with one of the accepted values which are `snapshot-controller`, `snapshot-webhook`, or `none`. `none` is mutually exclusive with all other values.
+* Specifying `optionalCsiComponents` with one of the accepted values which are `snapshot-controller`, or `none`. A value of `none` is mutually exclusive with all other values.
```

Comment:
```suggestion
* Specifying `optionalCsiComponents` with one of the accepted values which are `snapshot-controller`, or `none`. An entry of `none` is mutually exclusive with all other values.
```
Same as https://github.com/openshift/openshift-docs/pull/87662/files#diff-5fdd35dcb5f1bf5c78425d5ba6f3bedbf137bef6b93963d86e944b78a7a0e7cbR262 to be consistent with the same wording.